src/BARON.jl

```diff
@@ -1,11 +1,11 @@
-__precompile__()
```

Comment:
This is the default on Julia 0.7+ (https://github.com/JuliaLang/julia/pull/26991)